libs/auth/src/selectors/auth.selector.ts

```diff
@@ -0,0 +1,49 @@
+import { createSelector, MemoizedSelector } from '@ngrx/store';
+
+import { getDaffAuthFeatureStateSelector } from './auth-feature.selector';
+import { DaffAuthToken } from '../models/auth-token';
+import {
+  DaffAuthReducerState
+} from '../reducers/public_api';
+import { DaffAuthRegisterSelectors, getDaffAuthRegisterSelectors } from './register.selector';
+import { DaffAuthLoginSelectors, getDaffAuthLoginSelectors } from './login.selector';
+
+
+export interface DaffAuthSelectors<T extends DaffAuthToken> extends DaffAuthRegisterSelectors, DaffAuthLoginSelectors<T> {
+  selectAuthState: MemoizedSelector<object, DaffAuthReducerState>;
+  selectAuthLoading: MemoizedSelector<object, boolean>;
+  selectAuthErrors: MemoizedSelector<object, string[]>;
+}
+
+const createAuthSelectors = <T extends DaffAuthToken>() => {
+  const selectAuthState = createSelector(
+    getDaffAuthFeatureStateSelector<T>(),
+    state => state.auth
+  )
+
+  const selectAuthLoading = createSelector(
```

Comment:
I think that these selectors should go in a different file too. I think we should only have selectors in the main file when we don't have a choice, i.e. when they are selectors that depend on other selectors within auth.

Reply:
added a new file to export all the selectors and only exported that new file